### DISCUSSION TITLE
Cannot install

### DISCUSSION THREAD

**Discussion by BrandonHextall** [tribalfs/GalaxyMaxHzPub#168] [Q&A]
I am unable to install the all on one ui 6. I have tried all methods provided (including directly installing by pressing on the apk). Any help would be absolutely amazing

**Comment by Xisrr1** [comment]
Did you read the guide? You need an adb command.

**Comment by BrandonHextall** [comment]
Yeah I used the adb command. It refused to install

**Comment by Xisrr1** [comment]
Can you share more about the 'refused to install'? It is supposed to work.

**Reply by BrandonHextall** [reply]
So I tried running the adb command provided using LADB (cuz its the only thing I could use) and it ran into an error saying the file couldn't be found even though I tried every single possible combination for the apk path. Also tried just directly installing it using the Samsung installer. 

Also just had a thought when typing this. Does there need to be a "/" at the start before "internal storage"?

**Reply by Xisrr1** [reply]
You need to move the APK file to the root folder

**Reply by BrandonHextall** [reply]
So if I put the apk into the "internal storage" folder it should work?

**Reply by Xisrr1** [reply]
Yes

**Reply by BrandonHextall** [reply]
So I tried that. I ran it in LADB and used the command provided, along with different versions of "Internal storage/galaxy.apk". I tried using a / before internal storage, tried putting it in square brackets, tried putting it in quotation marks. Nothing. I dont get it. Is there something in missing?

**Reply by Xisrr1** [reply]
There is a video tutorial 

https://youtu.be/uWhj_q_YWDk

If this doesn't help I don't think I can do something, I'm not the dev.

**Reply by BrandonHextall** [reply]
Ok. Thanks mate

**Reply by Xisrr1** [reply]
Update me if it helped

**Reply by BrandonHextall** [reply]
Ok so its finding the file now. I used "internal storage" when it should be "sdcard" or something similar. But it now says "failed to open: permission denied". Any ideas?
Thank you for your hell btw it has made a big difference

**Reply by Xisrr1** [reply]
No, it should be "SD card", don't change the command at all.

**Reply by BrandonHextall** [reply]
Yeah I didn't change any of it I just couldn't remember what exactly it was. But yh is saying access denied for some reason and Idk why cuz realistically it should just work

**Reply by Xisrr1** [reply]
Damn

**Reply by BrandonHextall** [reply]
Any ideas?

**Reply by Xisrr1** [reply]
Nope. Try with a pc

**Reply by BrandonHextall** [reply]
Alrighty will do. Thanks again for your help I really appreciate the time taken

**Reply by Xisrr1** [reply]
No problem

**Comment by tribalfs** [comment]
DON'T SKIP reading the provided written instructions and ensure to use the latest LADB.

**Reply by BrandonHextall** [reply]
Ok so I got it installed. Everything is good. But the resolution option is greyed out and unusable. I turned off battery saver and it still doesn't do anything. Any help is greatly appreciated

**Reply by Xisrr1** [reply]
Read the guide.
